src/cheri_sys_regs.sail

```diff
@@ -5,6 +5,8 @@
 bitfield ccsr : xlenbits = {
   cap_idx : 15 .. 10,
   cause   :  9 .. 5, /* cap cause */
+  gclgu   :  3,      /* Global Cap Load Gen bit for User pages */
+  gclgs   :  2,      /* Global Cap Load Gen bit for System (not User) pages */
```

Comment:
It would seem this adds the bits per privilege level, rather than only for sccsr

Reply:
Also, it seems I can't comment on lines that weren't actually changed, but the legalize_ccsr function probably needs updating to allow changes to these bits.
I'm also surprised that sail still does exception cause/reg in CCSR (@jrtc27 should I go through and make that change?)